apps/mull-ui/src/app/app.tsx

```diff
@@ -5,6 +5,8 @@ import './app.scss';
 import { environment } from '../environments/environment';
 import { Message } from '@mull/types';
 
```

Comment:
redundant spacing, not sure if this is caused by prettier

Reply:
Do you mean the space in `{ environment }` and `{ message }`? That is typically how imports are formatted in JavaScript

Reply:
nah line 7